### ISSUE TITLE
Notification not accurate to reality

### ISSUE THREAD

**Issue by BennyOptions** [balancednetwork/balanced-network-interface#1221]
See the screenshot and relevant [transaction](https://tracker.icon.community/transaction/0xf24a407e853de5b7b1eb9b152fd591a4fe7293779f69afe36bd9d601a203daa3). The transaction was for 32,862 ICX, but notification says 33k. Need the notification to pull data from the blockchain, not from the FE expectations. This is the result of the sandwich attacker fyi.

![Screenshot 2023-07-19 at 3.20.01 PM.png](https://images.zenhubusercontent.com/5f07ce5a8cf129080163e5a4/bc0edbc5-3a23-497d-a15b-6f7b771e460b)

**Comment by FezBox** [comment]
@hetfly is this still a valid bug?

**Comment by hetfly** [comment]
Yes, but it requires considerable effort. Notifications are populated based on the estimates prior to the transaction execution. This would require fetching transaction logs after the execution and updating the notification based on that, so refactoring whole notification system, as well.
